.rubocop.yml

```diff
@@ -8,10 +10,50 @@ AllCops:
     - 'spec/fixtures/**/*'
     - 'vendor/**/*'
   TargetRubyVersion: 2.7
+  NewCops: enable
 
+# Warn only for long lines
 Layout/LineLength:
   Max: 120
+  Severity: warning
 
+# Use single-quoted strings where possible
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
+
+# Prevent false positives because we have fields named "subject"
+RSpec/SubjectDeclaration:
+  Enabled: false
+
+RSpec/NamedSubject:
+  Enabled: false
+
+# Allow long test blocks with multiple expectations
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/ExampleLength:
+  Enabled: false
+
+# Warn only for complexity-related metrics
```

Comment:
we could disable these entirely...imo they're of limited value and hard to remedy. alternatively we could raise the thresholds for some of them